src/pages/search/components/NoSupport.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import FontAwesome from 'react-fontawesome';
+import { Row, Col } from 'react-bootstrap';
+
+function NoSupport() {
+  return (
+    <div className='flexWrapper'>
+      <div className='verticalAlign'>
+        <h3 className='colourDarkGrey text-center'>
+          This page is not supported in your browser
+        </h3>
+        <p className='colourDarkGrey'>
+          {
+            'Either upgrade to the latest version or download one of the ' +
```

Comment:
IE11 is the latest version of IE, but it doesn't support promises. How about "We suggest upgrading to the latest version of one of the browsers below"?

Reply:
I thought edge was the latest version of IE? I linked to the edge download page, so if someone was on an older version of IE they would likely click the edge logo.

Reply:
According to Microsoft, Edge and IE are different browsers. The logo's are similar enough that this might work, though. I'm still in favor of changing the copy.